src/python/dxpy/scripts/dx.py

```diff
@@ -2514,7 +2514,13 @@ def install(args):
 def uninstall(args):
     app_desc = get_app_from_path(args.app)
     if app_desc is None:
-        parser.exit(1, 'Could not find the app\n')
+        #in case we lost permission for this app, try to remove it by Id.
```

Comment:
Given that here we are trying to remove the app using its name, please consider rephrasing this comment as:
'# In case the user lost access to the app, try to remove it using the app's name'